data-structures/arrays/base-operations/flatten.js

```diff
@@ -7,4 +7,11 @@
  */
 export const flatten = (arr) => {
     //TODO
-};
\ No newline at end of file
+    var emptyArray = [];
+    var obj = {};
+    if(Array.isArray(arr) == false) {
+        return emptyArray;
+    } else {
+        return arr.flat(1);
```

Comment:
Do not use standard method flat please. I want to be sure You know how to implement it.